### ISSUE TITLE
library doesn't work with Node v.4.2.1

### ISSUE THREAD

**Issue by fponticelli** [devongovett/font-manager#4]
This is the first error I get:

```
In file included from ../src/FontManager.cc:5:
../node_modules/nan/nan.h:189:68: error: too many arguments to function call, expected at most 2, have 4
    return v8::Signature::New(v8::Isolate::GetCurrent(), receiver, argc, argv);
```

**Comment by sidorares** [comment]
+1
@devongovett  do you have plans/capacity to port it to node 4/5? I'm very keen to use it in https://github.com/sidorares/ntk

**Comment by sidorares** [comment]
looks like this fork works with modern node: https://github.com/princejwesley/font-manager

@princejwesley do you have plans for PR?

**Comment by princejwesley** [comment]
@sidorares Sure

**Comment by sidorares** [comment]
Thanks @fponticelli !

**Comment by devongovett** [comment]
Fixed in #5.
